update.php

```diff
@@ -1287,3 +1287,15 @@ function update_1514()
 
 	return Update::SUCCESS;
 }
+
+function update_1515()
+{
+	DI::config()
+	  ->beginTransaction()
+	  ->delete('config', 'hostname')
+	  ->delete('system', 'urlpath')
+	  ->delete('system', 'ssl_policy')
```

Comment:
Once again, I'm with @annando , we don't need to delete them if we don't use them anymore.